packages/shared/routes/login/views/EnterPin.svelte

```diff
@@ -153,29 +153,35 @@
     })
 </script>
 
-<div class="relative w-full h-full bg-white dark:bg-gray-900">
-    <button
-        data-label="back-button"
-        class="absolute top-12 left-5 disabled:opacity-50 cursor-pointer disabled:cursor-auto"
-        disabled={hasReachedMaxAttempts}
-        on:click={handleBackClick}
-    >
-        <div class="flex items-center space-x-3">
-            <Icon icon="arrow-left" classes="text-blue-500" />
-            <Text type="h5">{locale('general.profiles')}</Text>
-        </div>
-    </button>
-    <div class="pt-40 pb-16 flex w-full h-full flex-col items-center justify-between">
+<div class="w-full h-full bg-white dark:bg-gray-900">
+    <div class="flex w-full h-full justify-center items-center">
         <div class="w-96 flex flex-col flex-wrap items-center mb-20">
-            <Profile name={$activeProfile?.name} bgColor="blue" />
-            <Pin
-                bind:this={pinRef}
-                bind:value={pinCode}
-                classes="mt-10 {shake && 'animate-shake'}"
-                on:submit={onSubmit}
-                disabled={hasReachedMaxAttempts || isBusy}
-                autofocus
+            <Profile
+                name={$activeProfile?.name}
+                networkType={$activeProfile?.networkType || 'devnet'}
+                networkProtocol={$activeProfile?.networkProtocol || 'shimmer'}
```

Comment:
Same as in the EnterPin, can we make these enums on the right and change to `??`